### PR TITLE
Add notes for installing other Python versions

### PR DESCRIPTION
**Pull request by joesiewert**

### CODE REVIEW

File: _basic/languages-frameworks/python.md [hunk 1]
```diff
@@ -8,19 +8,22 @@ category: Languages &amp; Frameworks
...
-As the virtual environment from `${HOME}/.virtualenv` is already activated and configured for use on the build VMs, if you want to switch to version 3, simply add the following commands to your _Setup Steps_.
+As the virtual environment from `${HOME}/.virtualenv` is already activated and configured for use on the build VMs, if you want to switch to version 3, simply add the following commands to your _Setup Commands_.
 
 ```shell
 virtualenv -p $(which python3) "${HOME}/cache/python3_env"
 source "${HOME}/cache/python3_env/bin/activate"
 ```
+
+## Additional Versions
+If you need a specific Python version installed, use this [script](https://github.com/codeship/scripts/blob/master/languages/python.sh) in your _Setup Commands_.
```
Comment (mlocher) [inline _basic/languages-frameworks/python.md]: I would add a complete example for a different version of python. Main reason being that users don't copy/paste the script itself into their repository (or their setup commands), but use the `curl` command.

File: _basic/languages-frameworks/python.md [hunk 2]
```diff
@@ -8,19 +8,22 @@ category: Languages &amp; Frameworks
...
-You can use pip to install any dependencies, for example
+You can use **pip** to install any dependencies, for example:
 
 ```shell
 pip install -r requirements.txt
 ```
 
 ## Python 3
-As the virtual environment from `${HOME}/.virtualenv` is already activated and configured for use on the build VMs, if you want to switch to version 3, simply add the following commands to your _Setup Steps_.
+As the virtual environment from `${HOME}/.virtualenv` is already activated and configured for use on the build VMs, if you want to switch to version 3, simply add the following commands to your _Setup Commands_.
```
Comment (mlocher) [inline _basic/languages-frameworks/python.md]: Can we extend the first sentence to read:

> ... is already activated and configured for use _with Python 2_ on the build VMs,...

**Comment by mlocher** [comment]
@joesiewert could you follow up with the changes? Once those two comments are addressed this is good to be merged.

**Comment by joesiewert** [comment]
All set @mlocher.

**Pull request merged**